### PR TITLE
Merge Anirudh's auto expansion and add latex processing for dynamic hints

### PR DESCRIPTION
**Pull request by Sharonfish**

### CODE REVIEW

File: package.json
Comment (shreyabhandari0220) [inline package.json:1]: "homepage": "OATutor-DynamicHint-Support" should be changed back to "homepage": "place-holder"

File: README.md
Comment (shreyabhandari0220) [inline README.md:1]: Change git clone --recurse-submodules https://github.com/CAHLR/OATutor-DynamicHint-Support.git back to git clone --recurse-submodules https://github.com/CAHLR/OATutor.git

**Review by shreyabhandari0220 — CHANGES_REQUESTED** [review]
README.md - 
Change git clone --recurse-submodules https://github.com/CAHLR/OATutor-DynamicHint-Support.git back to git clone --recurse-submodules https://github.com/CAHLR/OATutor.git

package.json -
"homepage": "OATutor-DynamicHint-Support" should be changed back to "homepage": "place-holder"

File: README.md
Comment (Sharonfish) [reply README.md:1]: Got it! Thanks for looking into this!

File: package.json
Comment (Sharonfish) [reply package.json:1]: Thanks for this!

**Review by Sharonfish — COMMENTED** [review]
Revisions based on Shreya's comments.

**Review by shreyabhandari0220 — APPROVED** [review]
Great!

**Pull request merged**